### ISSUE TITLE
Add an option to hide the factions being played from a game

### ISSUE THREAD

**Issue by cryogen** [cryogen/throneteki#1525]

**Comment by DukeTax** [comment]
Not exactly sure what's meant by this but with the new game list I assume this can be closed?

**Comment by zacchiro** [comment]
@DukeTax not really, the point of this is allowing player to not have the factions/agendas they are playing visible in the game list.

An alternative to making this an option would be to impose that private games (and only private games) have hidden factions/agendas in the game list.
